src/components/shared/SourceView.js

```diff
@@ -91,7 +91,9 @@ export class SourceView extends React.PureComponent<SourceViewProps> {
       _mapGetKeyWithMaxValue(timingsForScrolling.totalLineHits) ??
       _mapGetKeyWithMaxValue(this.props.timings.totalLineHits);
     if (heaviestLine !== undefined) {
-      this._scrollToLine(heaviestLine - 5);
+      // Make sure the line is at least 1.
+      const lineToScroll = Math.max(1, heaviestLine - 5);
```

Comment:
This is the only behavior change. By chance I picked "5" as the line number in my test, and I got an error from codemirror.

Reply:
We could also move this check into `_scrollToLine`. Up to you.